src/callbacks.jl

```diff
@@ -131,7 +130,7 @@ function dss_callback(integrator)
     Y = integrator.u
     ghost_buffer = integrator.p.ghost_buffer
     if !ghost_buffer.skip_dss
-        @nvtx "dss callback" color = colorant"yellow" begin
+        NVTX.@range "dss callback" color = colorant"yellow" begin
```

Comment:
Why is this change needed?

Reply:
The @nvtx macro is defined in example (https://github.com/CliMA/ClimaAtmos.jl/blob/main/examples/hybrid/nvtx.jl) and cannot be used in src. But it is essentially the same as NVTX.@range.